scipy/special/_special_ufuncs.cpp

```diff
@@ -413,6 +415,12 @@ PyMODINIT_FUNC PyInit__special_ufuncs() {
     );
     PyModule_AddObjectRef(_special_ufuncs, "iv", iv);
 
+    PyObject *iv_ratio = SpecFun_NewUFunc(
+        {static_cast<func_dd_d_t>(special::iv_ratio)},
+        "_iv_ratio", iv_ratio_doc
+    );
```

Comment:
If you use the new infrastructure, you have to add the `float` loops too, not just `double`. You'll also need to create an overloaded version of the scalar kernel which takes `floats`, casts to double before calling the function, and then casts back to float when returning.

Reply:
Noted thanks.